edk2basetools/VfrCompiler/IfrUtility.py

```diff
@@ -0,0 +1,2238 @@
+import ctypes
+from edk2basetools.VfrCompiler.IfrError import *
+from edk2basetools.VfrCompiler.IfrCtypes import *
+from edk2basetools.VfrCompiler.IfrCommon import *
+from abc import ABCMeta, abstractmethod
+from edk2basetools.Common.LongFilePathSupport import LongFilePath
+
+VFR_PACK_SHOW = 0x02
+VFR_PACK_ASSIGN = 0x01
+VFR_PACK_PUSH = 0x04
+VFR_PACK_POP = 0x08
+DEFAULT_PACK_ALIGN = 0x8
+DEFAULT_ALIGN = 1
+MAX_NAME_LEN = 64
+MAX_BIT_WIDTH = 32
+EFI_VARSTORE_ID_MAX = 0xFFFF
+EFI_BITS_SHIFT_PER_UINT32 = 0x5
+EFI_BITS_PER_UINT32 = 1 << EFI_BITS_SHIFT_PER_UINT32
+EFI_FREE_VARSTORE_ID_BITMAP_SIZE = int((EFI_VARSTORE_ID_MAX + 1) / EFI_BITS_PER_UINT32)
+
+
+class SVfrPackStackNode(object):
+    def __init__(self, Identifier, Number):
+        self.Identifier = Identifier
+        self.Number = Number
+        self.Next = None
+
+    def Match(self, Identifier):
+        if Identifier == None:
+            return True
+        elif self.Identifier == None:
+            return False
+        elif self.Identifier == Identifier:
```

Comment:
Just return the identifier comparison


```suggestion
        return self.Identifier == Identifier
```